docs/schedulers.adoc

```diff
@@ -0,0 +1,110 @@
+
+== Schedulers
+
+=== `fixed_priority_scheduler`
+
+A `fixed_priority_scheduler` represents work that will be run at a certain priority.
+
+[source,cpp]
+----
+using S = async::fixed_fixed_priority_scheduler<0>; // highest priority
+----
+
+NOTE: The intended use case for `fixed_priority_scheduler` is to schedule tasks
+to be executed on prioritized interrupts.
+
+The `fixed_priority_scheduler` works hand in glove with a `task_manager` that
+manages tasks in priority order. A `priority_task_manager` is provided and may
```

Comment:
suggestion: might want to reference "Rate Monotonic Scheduling" https://en.wikipedia.org/wiki/Rate-monotonic_scheduling

Reply:
Sure - haven't added time-priority mapping yet.